python/pyspark/pandas/tests/connect/plot/test_parity_series_plot_matplotlib.py

```diff
@@ -24,6 +24,10 @@
 class SeriesPlotMatplotlibParityTests(
     SeriesPlotMatplotlibTestsMixin, PandasOnSparkTestUtils, TestUtils, ReusedConnectTestCase
 ):
+    @unittest.skip("Test depends on Spark ML which is not supported from Spark Connect.")
```

Comment:
it failed with `"Empty 'DataFrame': no numeric data to plot"` before, now fails with `PandasNotImplementedError`